src/app/content/components/Page/searchHighlightManager.ts

```diff
@@ -81,11 +82,12 @@ const handleUpdate = (services: Services) => (
   selectResult(services, previous, current, options);
 };
 
-const searchHighlightManager = (container: HTMLElement) => {
+const searchHighlightManager = (container: HTMLElement, intl: IntlShape) => {
   const services = {
     container,
     highlighter: new Highlighter(container, {
       className: 'search-highlight',
+      formatMessage: ({ id }) => intl.formatMessage({ id }, { style: 'search' }),
```

Comment:
Could you give some explanation about `{ style: 'search' }`?

Reply:
`Highlighter` will call `formatMessage` with `{id: string, style: IHighlightData['style'] | undefined}`
This mean that `style` is basically set by us, and normal highlights have styles like `yellow`, `green`, etc. but search highlight doesn't have `style` property and we need it to create text https://github.com/openstax/rex-web/pull/1107/files#diff-51e75076ae4cd9a4653ce8a1403c42dab5790b384f89645d40abb917e68dde80R36

The `id` is created in `Highlighter`: https://github.com/openstax/highlighter/blob/master/src/injectHighlightWrappers.ts#L68

Reply:
Ok, I see. Thanks for the explanation.